### PR TITLE
Add support for remaining IPStackRef functions

### PR DESCRIPTION
**Pull request by astarche**
### What does this Pull Request accomplish?

Add support for remaining `IPStackRef` functions to the `nixnetsocket` service.

### Why should this Pull Request be merged?

Round out functionality in grpc-device service.

### What testing has been done?

Ran and passed all Xnet system tests and all unit tests.

### CODE REVIEW

File: generated/nixnetsocket/nixnetsocket.proto [hunk 1]
```diff
@@ -241,6 +246,34 @@ message ConnectResponse {
   int32 error_num = 3;
 }
 
+message GetNameInfoRequest {
+  nidevice_grpc.Session stack_ref = 1;
+  SockAddr addr = 2;
+  int32 host_len = 3;
+  int32 serv_len = 4;
+  int32 flags = 5;
```
Comment (astarche) [inline generated/nixnetsocket/nixnetsocket.proto:254]: TODO: use enum.

Comment (astarche) [reply generated/nixnetsocket/nixnetsocket.proto:254]: Actually I should revert this. I meant to accept Ryan's version.

**Comment by astarche** [comment]
image

**Review by reckenro — APPROVED** [review]

File: source/codegen/metadata/nixnetsocket/functions.py [hunk 2]
```diff
@@ -85,6 +85,29 @@
         ],
         'returns': 'int32_t'
     },
+    'InetAddr': {
```
[the remark this answers is not in the document]
Comment (astarche) [reply source/codegen/metadata/nixnetsocket/functions.py:88]: Created AB#1896764

**Comment by astarche** [comment]
@SparkingSpork Sorry, I accidentally pushed a cmakelists change in a previous rev. I don't think there's anything for you in this change.

**Review by mgumble — APPROVED** [review]

File: generated/nixnetsocket/nixnetsocket.proto (hunk 1 above)
Comment (astarche) [reply generated/nixnetsocket/nixnetsocket.proto]: Abandoning this for now. Will take this up in other PRs.

**Pull request merged**